### DISCUSSION TITLE
Data picker

### DISCUSSION THREAD

**Discussion by akashramtohul4** [quasarframework/quasar#10136] [General - Components / Directives / etc]
Hello Friends 

Am try to create a date picker like an input field but its getting massed up like this
![image](https://user-images.githubusercontent.com/84211129/126335935-8d6ad494-6a68-4b37-aae3-1780a438e3bb.png)

can someone help to display a more elegant way for a date picker

Thanks

**Comment by MilosPaunovic** [comment]
A code sample of any kind would help in debugging this, but it seems that the height of your input field is the problem, as its squashing the label.
